bundles/org.openhab.binding.konnected/README.md

```diff
@@ -28,24 +28,33 @@ The blink setting will disable the transmission LED on the Konnected Alarm Panel
 
 ## Channels
 
-The auto discovered thing adds two default channels.
+Depending on whether the thing type is the wifi version or the pro version the automativally added channels will differ.  
+For the wifi verision the auto discovered thing adds two default channels.
 
 | Channel | Channel Id | Channel Type | Description                                              |
 |---------|------------|--------------|----------------------------------------------------------|
-| 1       | Zone_6     | Switch       | A Switch channel for zone 6                              |
-| 2       | Out        | Actuator     | The Channel for the Out Pin on the Konnected Alarm Panel |
+| 1       | Zone_6     | Switch-wifi       | A Switch channel for zone 6                              |
+| 2       | Out        | Actuator-wifi     | The Channel for the Out Pin on the Konnected Alarm Panel |
 
 One channel for Zone 6 which is a sensor type channel, and one channel for the out pin that is an actuator type channel.
 These channels represent the two pins on the Konnected Alarm Panel whose type cannot be changed.
-For zones 1-5, you will need to add channels for the remaining zones that you have connected and configure them with the appropriate configuration parameters for each channel.
 
+For the Konnected Alarm Panel Pro the auto discovered thing adds three default channels.
+
+| Channel | Channel Id | Channel Type | Description                                              |
+|---------|------------|--------------|----------------------------------------------------------|
+| 1       | Alarm1     | Switch-Pro       | 12v Alarm 1 Output on the Konnected Pro                      |
+| 2       | Output1    | Actuator-Pro     | 3.3v Output 1 on the Konnected Alarm Panel					 |
+| 3       | Alarm2-Output2  | Switch-Pro     | The Selectable 12v Alarm 2 / 3.3v Output 2 on the Konnected Alarm Panel Pro |
+
+For zones 1-5 (wifi version) or zones 1-12 (pro version), you will need to add channels for the remaining zones that you have connected and configure them with the appropriate configuration parameters for each channel.
```

Comment:
How do you add Channels via the UI? Or is this supposed to be configured only via text?

Reply:
Channels are added in the UI under the channels screen.  There is an add channels button and the channels can then be added to the thing.  This is inherent with an extensible channel.

Reply:
I never seen this. Thanks!